### PR TITLE
Bump YDeck to 0.2.0, remove TCG ANGU override, increase ygo-data preload to notify

### PR DESCRIPTION
**Pull request by kevinlul**
## Description

Fixes #

## Checklist

- [x] I am following the [contributing guidelines](https://github.com/AlphaKretin/emcee-tournament-bot/blob/master/.github/CONTRIBUTING.md).
- [x] I have updated any relevant [user guides](https://github.com/AlphaKretin/emcee-tournament-bot/blob/master/guides).
- [x] I have updated any relevant [documentation](https://github.com/AlphaKretin/emcee-tournament-bot/blob/master/docs).
- [x] I have updated the [changelog](https://github.com/AlphaKretin/emcee-tournament-bot/blob/master/changelog.md), or this change is not user-facing.

### CODE REVIEW

File: src/deck.ts [hunk 1]
```diff
@@ -174,13 +187,16 @@ export class DeckManager {
...
-	private parseDeckError(err: DeckError): string {
+	private formatDeckError(err: DeckError): string {
 		if (err.type === "size") {
 			return `${this.capFirst(err.target)} Deck too ${err.min ? "small" : "large"}! Should be at ${
 				err.min ? `least ${err.min}` : `most ${err.max}`
 			}, is ${err.actual}!`;
 		}
 		// else type is limit
-		return `Too many copies of card ${err.name} (${err.target})! Should be at most ${err.max}, is ${err.actual}!`;
+		const name = this.cardIndex.has(err.target)
```
Comment (AlphaKretin) [inline src/deck.ts:209]: We can use the name provided by ydeck post-restoration

File: src/deck.ts [hunk 2]
```diff
@@ -30,65 +29,82 @@ export function splitText(outString: string, cap = 2000): string[] {
...
 	}
-	return new Card(card.text.en.name, card.data.ot, card.data.type, card.data.setcode, statusMap);
+	return {
+		name: card.text.en.name,
+		type: card.data.type,
+		setcode: card.data.setcode,
+		alias: card.data.alias || undefined,
+		limitTCG,
+		limitOCG,
+		isPrerelease: !!(card.data.ot & 0x100)
```
Comment (AlphaKretin) [inline src/deck.ts:56]: We've got the ygoData enums included here so this could be `& enums.ot.OT_PRERELEASE`

Comment (kevinlul) [reply src/deck.ts:56]: That is in fact not a thing! 😂

Comment (AlphaKretin) [reply src/deck.ts:56]: Heck. Can't wait to get this whole thing cleaned up.

**Review by AlphaKretin — APPROVED** [review]

**Pull request merged**